### ISSUE TITLE
Log spam: Lower "allocatable:" log from error-level

### ISSUE THREAD

**Issue by DreadPirateShawn** [kubernetes/kubernetes#50167]
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
Kubelet emitting error-level logs in production environment, like so:
```
E0803 20:42:47.064138    1445 kubelet_resources.go:46] allocatable: map[pods:{{110 0} {<nil>} 110 DecimalSI} cpu:{{8 0} {<nil>} 8 DecimalSI} memory:{{67438174208 0} {<nil>}  BinarySI}]
```

**What you expected to happen**:

Looking at the relevant source code -- https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_resources.go -- this looks like a debug log output of normal, expected behavior, which is accidentally set to error level.

In our enterprise production environment, there's a QA / reliability team which monitors for any error-level logs, and expects that our engr can explain / track / fix / address them in some manner. If this truly is debug output, which it looks like, then changing it to anything-other-than-error level will be extremely appreciated.

**How to reproduce it (as minimally and precisely as possible)**:
n/a

**Anything else we need to know?**:
n/a

**Environment**:
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.5", GitCommit:"490c6f13df1cb6612e0993c4c14f2ff90f8cdbf3", GitTreeState:"clean", BuildDate:"2017-06-14T20:15:53Z", GoVersion:"go1.7.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.5", GitCommit:"490c6f13df1cb6612e0993c4c14f2ff90f8cdbf3", GitTreeState:"clean", BuildDate:"2017-06-14T20:03:38Z", GoVersion:"go1.7.6", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration**: bare metal
- OS (e.g. from /etc/os-release): Ubuntu 16.04.1 LTS
- Kernel (e.g. `uname -a`): `Linux tuk6m1knode1 4.4.0-83-generic #106-Ubuntu SMP Mon Jun 26 17:54:43 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
- Install tools: n/a
- Others: n/a

**Comment by yujuhong** [comment]
> Looking at the relevant source code -- https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_resources.go -- this looks like a debug log output of normal, expected behavior, which is accidentally set to error level.

Agreed.

/cc @vishh @dashpole 
/sig node

**Comment by FengyunPan** [comment]
@DreadPirateShawn I also met for this problem, and I find that it is caused by this PR:
https://github.com/kubernetes/kubernetes/pull/41234
It is really a debug log:
https://github.com/kubernetes/kubernetes/pull/41234/files#diff-5f3d63d8d65184fcc8fbae4de504db66L43
We should change its level, "Infof" will be better, and I files a pr soon, please take a look.

**Comment by dims** [comment]
/close

Looks like this was fixed in #50181
